lib/prompts/createApiSamplePrompt.ts

```diff
@@ -1,63 +1,90 @@
-// @ts-nocheck
-const { promptUser } = require('./promptUtils');
-const { i18n } = require('../lang');
+import { promptUser } from './promptUtils';
+import { i18n } from '../lang';
+import { PromptOperand, PromptConfig } from '../../types/prompts';
 
 const i18nKey = 'lib.prompts.createApiSamplePrompt';
 
-const getSampleTypesPrompt = choices => ({
-  type: 'rawlist',
-  name: 'sampleType',
-  message: i18n(`${i18nKey}.selectApiSampleApp`),
-  choices: choices.map(choice => ({
-    name: `${choice.name} - ${choice.description}`,
-    value: choice.id,
-  })),
-  validate: input => {
-    return new Promise(function(resolve, reject) {
-      if (input.length > 0) {
-        resolve(true);
-      }
-      reject(i18n(`${i18nKey}.errors.apiSampleAppRequired`));
-    });
-  },
-});
+type SampleChoice = {
+  name: string;
+  description: string;
+  id: string;
+  languages: string[];
+};
+
+type SampleConfig = {
+  samples: SampleChoice[];
+};
+
+type CreateApiSamplePromptResponse = {
+  sampleType?: string;
+  sampleLanguage?: string;
+};
+
+function getSampleTypesPrompt(
+  choices: SampleChoice[]
+): PromptConfig<CreateApiSamplePromptResponse> {
+  return {
+    type: 'rawlist',
+    name: 'sampleType',
+    message: i18n(`${i18nKey}.selectApiSampleApp`),
+    choices: choices.map(choice => ({
+      name: `${choice.name} - ${choice.description}`,
+      value: choice.id,
+    })),
+    validate: function(input?: string) {
+      return new Promise<PromptOperand>(function(resolve, reject) {
```

Comment:
I saw your comment saying you addressed this one but it looks like maybe that change didn't go through. This should be `boolean`, not `PromptOperand` (and then `PromptOperand` doesn't need to be exported

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1288/commits/d73f2185f196a582671543443e00908845022b7c